plugins/legacy-plugin-chart-chord/src/ReactChord.jsx

```diff
@@ -16,7 +16,44 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import React from 'react';
 import { reactify } from '@superset-ui/chart';
+import styled from '@superset-ui/style';
+import PropTypes from 'prop-types';
 import Component from './Chord';
 
-export default reactify(Component);
+const ReactComponent = reactify(Component);
+
+const Chord = ({ className, ...otherProps }) => {
+  return (
+    <div className={className}>
+      <ReactComponent {...otherProps} />
+    </div>
+  );
+};
+
+Chord.defaultProps = {
+  otherProps: {},
+};
+
+Chord.propTypes = {
+  className: PropTypes.string.isRequired,
+  otherProps: PropTypes.objectOf(PropTypes.any),
+};
+
+export default styled(Chord)`
+  .superset-legacy-chart-chord svg #circle circle {
```

Comment:
Just a note for all of these plugins: Emotion uses SASS, so you can condense these styles a bit, e.g.

```.superset-legacy-chart-chord svg{
  #circle {
    circle {
      fill: none;
      pointer-events: all;
    }
    &:hover path.fade {
      opacity: 0.2;
    }
  } 
  .group path {
    fill-opacity: 0.6;
  }
  path.chord {
    stroke: #000;
    stroke-width: 0.25px;
  }
}```

And I _think_ you might not even need the `.superset-legacy-chart-chord` there, since the CSS should be scoped to the component.